src/django/api/views.py

```diff
@@ -216,14 +215,18 @@ def create(self, request):
         header = csv_file.readline().decode().rstrip()
         self._validate_header(header)
 
-        # TODO: Get the organization from the authenticated user
-        organization = Organization.objects.first()
+        organization = Organization.objects.get(name=request.user.name)
```

Comment:
I ran into a not found exception on this line. The `request.user` object has an `organization` property that should be used here. It's optional, so it needs to be checked for non-None. (Fixing this would take care of #105)

Reply:
Good catch! I'll add this check.

Reply:
Added in 
15c572b along with a test.